bokehjs/test/models/formatters/func_tick_formatter.coffee

```diff
@@ -34,7 +34,7 @@ describe "func_tick_formatter module", ->
 
     it "should support imports using require", ->
       formatter = new FuncTickFormatter({
-        code: "var _ = require('underscore'); return _.max([1,2,3])"
+        code: "var max = require('../../core/util/array').max; return max([1, 2, 3])"
```

Comment:
This `../..` seems unnecessary, but otherwise tests fail. This is because this import isn't resolved as in custom models, but relative to `FuctTickFormatter` model (as if we were developing bokehjs here).